### DISCUSSION TITLE
FSR Launcher Feature

### DISCUSSION THREAD

**Discussion by RyzenAdvanced** [Blinue/Magpie#179] [Ideas]
Hello Devs,

Would be nice to see "FSR Launcher" similar to Proton FSR in Linux, just for Windows.

The work-flow:

1. Magpie > add new game > select FSR Quality Preset > Apply
2. Each added game will be now listed in Magpie "Game Launcher"
3. Magpie will automatically open the game in the right resolution/scale/full screen optimized with FSR at the selected quality preset.
4. You can go to Magpie, select any game, select quality and click "Run".

Could be very nice!

**Comment by Blinue** [comment]
Thanks for the idea. There are difficulties in making Magpie into a launcher.
1. some games have their own launchers, and Magpie can't launch them directly
2. not all games support resizing, and there's no way to launch a game at a specified resolution

Some features have been discussed elsewhere:

**Auto resize window**: #173
**Configurations for specific games**: https://github.com/Blinue/Magpie/issues/166#issuecomment-939413505

**Comment by saberbholt** [comment]
Mmmm and a launcher as parameters in a bach file from terminal or cmd?.
Can use a format same:  start Magplie.exe [the game]  /optional game.ini (for the configuration of that especifi game)
Is a easy mode for use in front-ends in emulators and the most games, the only thing an be capture the id process launcher of the game and move the focus to it.
